src/commands/platforms/create.ts

```diff
@@ -69,6 +70,13 @@ export default class PlatformCreate extends Command {
     const created_platform = await this.post_platform_to_api(platform_dto, account.id);
     cli.action.stop();
 
+    this.log(`Hang tight! We're completing the setup of your platform by installing applications that will broker networking. This could take as long as 15m, so feel free to grab a cup of coffee while you wait.`);
+    this.log(`Logs here: ${this.app.config.app_host}/examples/platforms/new?platform_id=${created_platform.id}`);
+    cli.action.start(chalk.blue('Installing platform applications'));
```

Comment:
@davidthor I'd assume you want a confirmation here?

Reply:
Yes please. Just like in the UI.

Reply:
Should we support an `--auto-approve` flag here to skip the prompt?

Reply:
You can add, but its prob not going to be used often.

Reply:
Seconded. I don't see how/why platform registration would be a recurring, automated process. Environments will spin up and down, but the platform will only ever be registered once.